content/Simple dynamic settings for Django - Jorge Michelena.md

```diff
@@ -0,0 +1,100 @@
+## Simple dynamic settings for Django
+
+
+### Introduction
+
+Not long ago I finished working on a small project in Django, and I noted that there were a few functionalities depending directly on constant values defined in the *settings* file but that given the nature of said functionalities it seemed like a good idea to be able to modify those values to alter the application behaviour without having to edit the settings and deploy the project again.
```

Comment:
I would change `noted` by `noticed`.
Also I would do some change to add some punctuation to this sentence. I think is too long and needs some pause somewhere.